mindsdb/integrations/handlers/merlion_handler/setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
```

Comment:
Just realized this now, but you should the attributes below from `__about__`, because it is currently undefined.

Reply:
fixed

from .__about__ import __dict__ as about